docs/building-accessible-and-responsive-progressive-web-applications.md

```
@@ -668,7 +668,7 @@ This checklist provides a structured framework for conducting a comprehensive qu
  | Page provides a custom offline fallback experience. | Manual | DevTools Offline Mode |
 | **Perceivable** | All non-decorative images have descriptive `alt` text. | Manual | Code Review, Screen Reader |
 |\
- | Color contrast for text and UI components meets WCAG AA levels. | Automated | Lighthouse, DevTools Inspector |
+ | Colour contrast for text and UI components meets WCAG AA levels. | Automated | Lighthouse, DevTools Inspector |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)